op-chain-ops/cmd/check-l2/main.go

```diff
@@ -276,6 +276,99 @@ func checkPredeployConfig(client *ethclient.Client, name string) error {
 			if err := checkEAS(p, client); err != nil {
 				return fmt.Errorf("EAS: %w", err)
 			}
+
```

Comment:
It would be ideal if you could add tests for the code existing at the addresses in the solidity file. The best way to do this would be to add variables in `Predeploys.sol` for each new predeploy and then assert `address.code.length > 0`

Reply:
The idea is that we can delete `check-l2` and keep all solidity testing unified in one location instead of leaking it into other locations